src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/MediaControllerPostAdminMediasSearchIntTest.java

```diff
@@ -465,13 +468,84 @@ void adminSearchForMedia_ShouldReturnNoPermissionsException() throws Exception {
 
     }
 
+    @Test
```

Comment:
Why do we need a new test? Could an existing test not assert the order? (just thinking of trying not to add extra integration tests if we don't need them)